### ISSUE TITLE
Adding a Custom validator throw an error

### ISSUE THREAD

**Issue by pascualm4** [proengsoft/laravel-jsvalidation#175]
Hi. I'm adding a easy custom validation to my FormRequest but the JsValidator send an error message and doesn't make the validation correctly.

This is my FormRequest Class

```php
<?php

namespace App\Http\Requests;

use App\Http\Requests\Request;

class ConsumableAddRequest extends Request
{
    /**
     * Determine if the user is authorized to make this request.
     *
     * @return bool
     */
    public function authorize()
    {
        return true;
    }

/**
 * Get the validation rules that apply to the request.
 *
 * @return array
 */
public function rules()
{
    \Validator::extend('unique_test', function($attribute, $value, $parameters)
    {
        return $value == 'guantes' ? true : false;


    }, trans('validation.unique'));

    return [
        'description' => 'unique_test:consumables',

    ];
}

}
```

In the other validations (not custom validations) works well. Thanks in advance

**Comment by sakalauskas** [comment]
You should extend the Validator instance in Service provider, not in the request.

https://laravel.com/docs/5.2/validation#custom-validation-rules

**Comment by torrentalle** [comment]
This is fixed in release v1.4.3
